app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
"Ruby on Rails Tutorial Sample App" những chuỗi text hiển thị lên view của end user như này thì tìm hiểu I18n trong rails để đưa vào locale quốc tế hóa em nha
https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module